src/components/container/LabResultsBloodType/LabResultsBloodType.css

```diff
@@ -0,0 +1,53 @@
+.mdhui-lab-results-summary {
```

Comment:
Remove unused classes and update prefix to reflect this component. .mdhui-lab-results-blood-type

Reply:
fixed